html-forms/index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Forms</title>
+  </head>
+<body>
+  <h1>Coding Form!</h1>
+  <form action="" method="get" class="form-example">
+    <div class="form-example">
+      <label for="email">Email</label>
+      <div><input type="email" name="email" /></div>
+      <label for="cool">Why is coding cool?</label>
+    <div><select name="options">
+      <option value="option">Select an option</option>
+      <option value="interesting">Because it's interesting</option>
+      <option value="fun">Because it's fun</option>
+      <option value="powerful">Because it's powerful</option>
+    </select></div>
+      <label for="choose">Favorite Language (choose one)</label>
+    <div><radio>
+      <input type="radio" name="language" value="html" checked="checked" /> HTML
+      <input type="radio" name="language" value="css" /> CSS
+      <input type="radio" name="language" value="javascript" /> JavaScript
+    </radio></div>
+    <label for="signup">Would you like to sign up for more information?</label>
+    <div><checkbox>
+      <input type="checkbox" name="more" value="signup" /> Sign Me Up!
+    </checkbox></div>
+    <label for="comments">Comments</label>
```

Comment:
This `for` attribute here is correct, but the form control element it is paired with needs a matching `id` attribute with the same value.